SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -104,6 +104,12 @@ USpatialNetDriver::USpatialNetDriver(const FObjectInitializer& ObjectInitializer
 	// TODO: UNR-2375
 	bMaySendProperties = true;
 
+#if ENGINE_MINOR_VERSION > 25
```

Comment:
Minor, how come `>25` here and `>= 26` elsewhere?

Reply:
Good point, will making it consistently `>= 26`.